index.js

```diff
@@ -68,8 +68,14 @@ cli.on('require', function(name) {
   log.info('Requiring external module', ansi.magenta(name));
 });
 
-cli.on('requireFail', function(name) {
-  log.warn(ansi.yellow('Failed to load external module'), ansi.magenta(name));
+cli.on('requireFail', function(name, error) {
+  log.warn(
+    ansi.yellow('Failed to load external module'),
+    ansi.yellow(name)
```

Comment:
Sorry, my explanation might be bad. Please modify only a module name  to magenta as is.